bin/ttree

```diff
@@ -360,22 +362,34 @@ sub process_file {
     $dest = $destdir ? "$destdir/$destfile" : $destfile;
                    
 #    print "proc $file => $dest\n";
-    
-    # check against link list
-    foreach my $link_pattern (@$link) {
-        if ($filename =~ /$link_pattern/) {
-            $link_file = $copy_file = 1;
-            $check = $link_pattern;
-            last;
-        }
+
+    unless ($link_file) {
```

Comment:
this line seems unnecessary change? as `$link_file` is initialize to 0

Reply:
I put it in intentionally to make the code segments symmetric. This makes it easy to change the order, e.g. have assets detection first, link/copy later. 

I like my code to be flexible/maintainable.

Reply:
If you like the semantics of `assets` I can massage the code. Slightly.

Reply:
indeed this was more a style comment than anything else. this is fine
I'm not sure what you have in mind by `massage the code` but feel free to submit more PRs.
thanks for your help